identity/webapp/package.json

```diff
@@ -65,11 +62,9 @@
     "@typescript-eslint/eslint-plugin": "^5.54.1",
     "@typescript-eslint/parser": "^5.48.2",
     "babel-core": "^6.26.3",
-    "babel-jest": "^29.5.0",
     "babel-plugin-styled-components": "^2.1.4",
     "babel-plugin-transform-typescript-metadata": "^0.3.1",
     "babel-preset-react": "^6.24.1",
-    "jest": "^29.5.0",
     "msw": "^0.27.0",
     "nodemon": "^2.0.15",
     "ts-jest": "29.1.0",
```

Comment:
For some reason that one is still needed on here, which made me pause and ask around, if anyone has an idea of why.